Dockerfile

```diff
@@ -1,15 +1,16 @@
-FROM debian:trixie-slim
+FROM alpine:3.22
```

Comment:
there was a push from security to move away from alpine, that is why I went for Debian. I am fine with the change if it is still a soft recommendation and not a strong one 😄

Reply:
Mmm, I'll try to install `ash` then in the image.